New icon for PowerShell Core

SteveL-MSFT opened the pull request:
Fix https://github.com/PowerShell/PowerShell/issues/4676

Default color using familiar console icon, but with avatar (Posh-chan) colors

![icon](https://raw.githubusercontent.com/SteveL-MSFT/PowerShell/new-icon/assets/av_colors_128.svg?sanitize=true)
<!--

If you are a PowerShell Team member, please make sure you choose the Reviewer(s) and Assignee for your PR.
If you are not from the PowerShell Team, you can leave the fields blank and the Maintainers will choose them for you. If you are familiar with the team, feel free to mention some Reviewers yourself.

For more information about the roles of Reviewer and Assignee, refer to [CONTRIBUTING.md](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md).

-->

SteveL-MSFT committed:
added new icon

SteveL-MSFT committed:
added new icon

SteveL-MSFT committed:
update icon to new version of console image with avatar colors

SteveL-MSFT commented:
Vote here https://github.com/PowerShell/PowerShell/issues/4676#issuecomment-332377214

SteveL-MSFT committed:
added svg for blue Windows PowerShell icon

SteveL-MSFT committed:
changed default to the black icon

SteveL-MSFT committed:
added custom bitmaps to the installer

SteveL-MSFT commented:
Latest change is to add some customer bitmaps to the installer:

![installer](https://imgur.com/hXpkmb8.png)
![page2](https://imgur.com/3tCTo9r.png)

jpsnover commented:
Looks great!

iSazonov commented:
Can we make this woman's face beautiful? Only I think it skewed?

SteveL-MSFT commented:
@iSazonov are you referring to the "mouth"?  that's the cursor of the prompt, sort of like `>_`

iSazonov commented:
Yes,  "mouth", "nose" - I'd preffer see a nice face. Can we make from >_ a purse? :-)

SteveL-MSFT commented:
@iSazonov I think it's too late to make changes to Posh-chan.  It came from a pro designer and it's what we've been using in some marketing.  I'm just reusing one of the original assets.  Think of it more as "war paint" on her face ;P

SteveL-MSFT committed:
fixed `launch powershell` checkbox so it shows up more nicely as the default checkbox
has a grey background

also reformatted the xml using a xml formatter to make it easier to read

SteveL-MSFT commented:
Last change is to fix the `Launch PowerShell` checkbox as the default control has a grey background which overlaps with the black icon.  Now it looks like

![launch](https://imgur.com/ePdEEA1.png)

adityapatwardhan commented on assets/Product.wxs:
```diff
@@ -1,12 +1,10 @@
-﻿<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-
+﻿<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi">
```
Unnecessary change?

SteveL-MSFT replied:
Might have accidentally hit Enter on that line.  Will fix.

adityapatwardhan reviewed (approved):
LGTM with minor comment.

SteveL-MSFT committed:
address PR feedback

DimaRazb commented:
She seems very aggressive, I don't like her very much. Designer could use a purse or raincoat I believe.

joeyaiello commented:
@SteveL-MSFT It's unclear to me which one is the default in this PR. Does it need to be updated to the black icon by default still? 

Otherwise, LGTM. Love the new MSI look-and-feel.

SteveL-MSFT commented:
@joeyaiello default is:

![blackicon](https://raw.githubusercontent.com/SteveL-MSFT/PowerShell/new-icon/assets/ps_black_128.svg?sanitize=true)

SteveL-MSFT commented:
@PowerShell/powershell-committee reviewed this and fine with the changes

adityapatwardhan commented:
@joeyaiello Can you update your review?

joeyaiello reviewed (approved):
Yup, I'm good
